frontend/src/components/Footer/Footer.jsx

```diff
@@ -1,98 +1,71 @@
 import React from 'react';
-import { HashLink } from 'react-router-hash-link';
+import {
+  Footer as DSFooter,
+  FooterBody,
+  FooterBodyItem,
+  FooterBottom,
+  FooterLink,
+  Logo,
+  Link,
+} from '@dataesr/react-dsfr';
 
-import Logo from 'components/Logo/LogoFooter';
-import FooterBottomItem from './FooterBottomItem';
+const footerBodyLinks = [
+  { title: 'Contactez-nous', link: '/mentions-legales' },
```

Comment:
shouldn't we have something to manage the target ? in a new tab or in the same ?